### PR TITLE
Create chart-2.1.4.d.ts

### PR DESCRIPTION
**Pull request by anuti**
case 1. Add a new type definition.
- [X] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [ ] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

Chart JS library has been updated to version 2 although no one has yet created the proper definitions files. For this reason I propose my file while being aware that still needs lots of work. In addition to the usual bug there are  shortcomings in the definition of many functions, as the official guide does not specify the parameter's types.

in the hope that this can help someone, more updates will come.

### CODE REVIEW

**Comment by vvakame** [comment]
you should update https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/chartjs/chart.d.ts directly. Do not add new file.

**Comment by anuti** [comment]
I've added a new file because I did not want "overwrite" the v1 version. My code is far from complete and the "old" v1 files is still widely used. This is my first time on github, so if you think update the old definition is a better idea I will do. This week I also want to fix the Travis ci build error and add the test file.

**Comment by mhegazy** [comment]
Thank you for your contribution. 

Can you please fix the failing Travis CI tests and update the PR!

**Comment by mhegazy** [comment]
@anuti 🔔

**Comment by anuti** [comment]
I Apologize for the wasted time, I'd a lot of extra work this weeks.

 I'll fix the the travis test today.

File: chartjs/chartjs.d.ts [hunk 1]
```diff
@@ -0,0 +1,381 @@
+// Type definitions for Chart.js
+// Project: https://github.com/nnnick/Chart.js
```
Comment (vvakame) [inline chartjs/chartjs.d.ts]: Is this right? Which do you create this definition for `chart.js` or `chartjs` ?

**Comment by anuti** [comment]
> Is this right? Which do you create this definition for chart.js or chartjs ?
Should be "chart.d.ts", right?

**Comment by mhegazy** [comment]
In general the name should match the name of the npm package if one exists. looks like the name of the package is `chart.js`. so the name of the folder/file should be `chart.js`

**Comment by jramsay** [comment]
@anuti:  Thanks for adding this type definition. It looks like the only thing remaining is to rename the folder & file to chart.js/chart.js.d.ts - can you please update the PR with this change?

**Pull request merged**